packages/perf/lib/ScreenTrace.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2016-present Invertase Limited & Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this library except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+let id = 0;
+
+export default class ScreenTrace {
+  constructor(native, identifier) {
+    this.native = native;
+    this._identifier = identifier;
+    this._id = id++;
+    this._started = false;
+    this._stopped = false;
+  }
+
+  start() {
+    if (this._started) {
+      return Promise.resolve(null);
+    }
+    this._started = true;
+
+    return this.native.startScreenTrace(this._id, this._identifier);
```

Comment:
I did not see an iOS implementation, so this will crash on iOS unless there is a platform check here.
I think it's best to platform-check and then Promise.reject "not supported on iOS", document the same in the index.d.ts
This will get people in the habit of checking for rejections, which should steer them into implementations that handle Android 8/8.1 rejections correctly as well

Reply:
Sounds good to me.

Reply:
fixed

Reply:
@mikehardy I am not seeing the @platform tag being pulled in from the JSDoc comments onto the site: https://react-native-firebase-43jdz2ws9-invertase.vercel.app/reference/perf/screentrace. s there something bugged with the docs website?

Reply:
I haven't seen that go through either, I usually put it in the method text like so https://rnfirebase.io/reference/app-distribution#checkForUpdate - works for me ?

Reply:
Oh I see